### PR TITLE
MBS-9901: Change Dailymotion cleanup to https

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-9901

Dailymotion itself redirects to https nowadays.

### CODE REVIEW

File: root/static/scripts/tests/Control/URLCleanup.js [hunk 1]
```diff
@@ -953,22 +953,22 @@ const {LINK_TYPES, cleanURL, guessType, validationRules} = require('../../edit/U
         },
         // Dailymotion
         {
-                             input_url: 'https://dailymotion.com/who-knows#uploads',
+                             input_url: 'http://dailymotion.com/who-knows#uploads',
```
Comment (yvanzo) [inline root/static/scripts/tests/Control/URLCleanup.js]: There is no more test with `https` input Dailymotion URL.

Comment (reosarevok) [reply root/static/scripts/tests/Control/URLCleanup.js]: Should there be? It's not like https is going to get cleaned up to anything but https. But I can change one back if desired :)

Comment (yvanzo) [reply root/static/scripts/tests/Control/URLCleanup.js]: Just making sure the it is still/will still be recognized by the system. This kind of regression happened in the past.

Comment (reosarevok) [reply root/static/scripts/tests/Control/URLCleanup.js]: Ok, changed one for this :)

**Pull request merged**